PostPairsTrading.md

```diff
@@ -0,0 +1,260 @@
+
+## Cointegration, then what?
+
+Let's recap our progress:
+
+1. We've acknowledged the randomness of the market, yet discovered that certain assets **exhibit similar movements**.
+
+2. By employing the **cointegration method and the ADF test**, we pinpointed a promising pair of assets for our analysis: NASDAQ100 and SP500.
+
+3. These two assets exhibit similar movements and **tend to gravitate around a shared mean**.
+
+Now we're faced with a crucial question: **"What do I do with these assets?"**
+
+As mentioned earlier, the market is inherently random and doesn't always behave predictably. While NASDAQ100 and SP500 often follow similar trends, their individual values **can sometimes diverge significantly**. For instance, NASDAQ100 may rise while SP500 falls, or vice versa. 
+
+However, this presents **an opportunity for profit** because we know that these assets tend to revert to their shared mean over time. If one asset is **overpriced** and likely to decrease, we may consider **selling it** (going short). Conversely, if an asset is **underpriced** and expected to increase, we may consider **buying it** (going long). And that is what we call Pairs Trading.
+
+> 💡 This strategy possesses intriguing financial characteristics: our **profitability remains unaffected by the broader market trends**, as our focus lies solely on the disparity between the two assets. It's about relative movements rather than absolute ones; we're indifferent to whether prices are rising or falling. This quality defines it as a **neutral market strategy**.
+
+## Spreading spreads
+
+Indeed, just subtracting the prices of two assets, as in: $priceY−priceX$ may not provide a clear understanding of their relationship. Let's illustrate this with an example:
+
+Consider the following series:
+
+---
+
+```q
+priceX: 5 10 7 4 8
+
+priceY: 23 30 25 30 35
+
+spreads: priceY - priceX // = 18 20 18 26 27
+```
+
+> 💡 As you can see and verify through Q/kdb+ array properties, spreads can be calculated by simply computing the difference between two vectors without the need for special functions or loops.
+
+---
+
+〽 **These spread values don't offer much insight** into the relationship between the two assets. Are both assets increasing? Are they moving in opposite directions? It's unclear from these numbers alone.
+
+Let's consider **using logarithms**, as they possess favorable properties for our pricing model. They inherently prevent negative values and tend to approach zero:
+
+--- 
+
+```q
+log priceX = 1.609438 2.302585 1.94591 1.386294 2.079442
+
+log priceY = 3.135494 3.401197 3.218876 3.401197 3.555348
+
+spreads: log priceY - log priceX // = 1.526056 1.098612 1.272966 2.014903 1.475907

```

Comment:
there is a bug here, you need the use of parentheses or brackets
```suggestion
spreads: log[priceY] - log priceX // = 1.526056 1.098612 1.272966 2.014903 1.475907

```
And i think is a good moment to explain that q has no more preference order than right to left